src/devices/CharacterLcd/samples/Pcf8574tSample.cs

```diff
@@ -0,0 +1,353 @@
+﻿using System;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+using System.Timers;
+using Iot.Device.Mcp23xxx;
+using Iot.Device.CharacterLcd;
+
+namespace Iot.Device.CharacterLcd.Samples
+{
+    class Pcf8574tSample
+    {
+        public class ExtendedSample
+        {
+            const string Twenty = "123456789\u0008123456789\u0009";

```

Comment:
Do you mind adding a comment here explaining what are these supposed to be?

Reply:
this is not my code; i copied it from Hd44780.ExtendedSample.cs into a new project to debug; the original author would be best placed to comment here